### ISSUE TITLE
Quickstart with Relay Modern should not use React Router 4

### ISSUE THREAD

**Issue by stevensacks** [graphql-boilerplates/react-fullstack-graphql#62]
This project currently uses React Router 4, however, the Relay Modern documentation says you should not use RR4 with it.

https://facebook.github.io/relay/docs/routing.html

> The component-based approach of React Router v4 does not readily allow for aggregating the data requirements for nested routes, and as such does not readily permit an approach that will avoid request waterfalls from nesting QueryRenderer components.

> **Found**
> Found offers integration with Relay Modern and Relay Classic via Found Relay. Found Relay runs queries for matched routes in parallel, and supports fetching Relay data in parallel with downloading async bundles from code splitting when using Relay Modern.

**Comment by marktani** [comment]
This issue is outdated. Please open a new issue if you have any problems or questions.
